### DISCUSSION TITLE
changing global tableNames using NamingStrategy in NestJs

### DISCUSSION THREAD

**Discussion by BAKHYESUNG** [mikro-orm/mikro-orm#3283] [Q&A] [answered]
Hi I tried to change toUpperCase all my tableNames but seems like it isnt working. 

```
import { EntityCaseNamingStrategy } from '@mikro-orm/core';    
export class CustomNamingStrategy extends EntityCaseNamingStrategy {    
  constructor() {
    super();
  }
classToTableName(entityName: string): string {
    return super.classToTableName(entityName).toUpperCase();
  }
}
```
here is my custom.naming.strategy.ts

and I've import my config via package.json both package.json and and place it right into module.forRoot({...NamingStratege:...}) is not working for me.

**Comment by BAKHYESUNG** [comment, marked as answer]
turns out it's because of the cached meta data in temp/{entitty}.json I deleted them and start it again it works fine. hope this works to anyone who had same problem as mine. thanks.

**Reply by B4nan** [reply]
Right, we cache the metadata based on the content of the entity definition (literally md5 hash of the file contents), so if you change something in the config, you need to wipe it first. 

I guess we could also take the ORM config into account when computing the hash. Worst case we will have some cache misses if you change things in the config, but it will be more robust in general.
